### ISSUE TITLE
REST implementation to power off the display

### ISSUE THREAD

**Issue by tringler** [thanksmister/wallpanel-android#37]
**Is your feature request related to a problem? Please describe.**
I would really like to have a REST option to power off the display. It is not a problem, but it would nice to have it, that i.e. I can power off the screen with my bulbs - Example HASS automation:

```
- alias: floor_light_turn_off_floorled_by_motion
  trigger:
    - platform: state
      entity_id: binary_sensor.motion_sensor_158d00016db094
      from: 'on'
      to: 'off'
    - platform: state
      entity_id: binary_sensor.motion_sensor_158d0001a24994
      from: 'on'
      to: 'off'
  condition:
    condition: and
    conditions:
      - condition: state
        entity_id: binary_sensor.motion_sensor_158d00016db094
        state: 'off'
      - condition: state
        entity_id: binary_sensor.motion_sensor_158d0001a24994
        state: 'off'
  action:
    - service: light.turn_off
      entity_id: light.gateway_light_286c0788b78a
    - service: switch.turn_off
      entity_id: switch.floor_wallpanel_wakeup
```

**Describe the solution you'd like**
I want to have a REST implementation to power off the display like `curl -H 'Content-Type: application/json' -X POST -d {wake:false} http://192.168.1.251:2971/api/command`

**Describe alternatives you've considered**
Waiting until the device turns off automatically

Thanks in Advance, Thomas

**Comment by ReX1983** [comment]
Similar request from me.

I am trying to keep the display on as long as movement is detected in a given room. 

If turn on the screen using the MQTT / REST screen on command, it stays on for some time but then it goes off. Android doesn't allow to change the screen on when in lock screen mode.

I tried to send screen on messages each 10-15 seconds but they get ignored if the screen is already on. So a keep alive solution doesn't really work.

I think the best solution for my need is to send a turn on screen message that keeps the display on till the next turn off message. As alternative I'd like screen on message to not get ignored when the screen is already on, so I can use them as keep alive.

**Comment by thanksmister** [comment]
Right now the screen is set to go back into screensaver mode after 30 seconds.  We are not continuously broadcasting the screen wake during continuous motion detection, but we could do this with some adjustments.   I didn't want to flood the MQTT client with constant messages that motion was detected, that is why I delay reacting to motion detected after the first time motion is detected.

**Comment by thanksmister** [comment]
"I would really like to have a REST option to power off the display"

You can not power off your screen, but you could set the screen brightness to 0 or close too it.  I am not sure if that is what you are referring to?

**Comment by allofmex** [comment]
Important for me too. Want to wake display when room occupied (external sensor) and switch off display if not occupied anymore. 
Thats currently impossible because I can just wake, then it will switch off after Androids display timeout. I can wake again only after it went off, but that's wired (periodically off/on/off/on...).

**Comment by thanksmister** [comment]
As I mentioned, you can't power off an Android screen or wake it once it is off.  You can change the brightness and use the screensaver option.  Those features are already present.   The only other thing you could do is quit the application with a command and then the Android device would sleep, but you can't relaunch it again.
